stix_shifter_modules/secretserver/stix_transmission/api_client.py

```diff
@@ -35,11 +36,15 @@ def get_token(self):
         response = RestApiClient.call_api(self, self.auth_token_url, 'GET', headers=self.headers, data=self.payload,
                                                                             urldata=None,
                                                                             timeout=None)
-        res = response.response.text
-        json_obj = json.loads(res)
-        token = json_obj.get('access_token')
-        self.accessToken = 'Bearer' + " " + token
-        return self.accessToken
+        if(response.code !=200):
+            logging.error("Login failed",response.code )
+            exit(0)
```

Comment:
you can't exit code like this. you need to raise the exception and make sure the error is returned in the error object by calling `ErrorResponder.fill_error()`